src/components/header.svelte

```diff
@@ -7,6 +7,7 @@
 
   let mobileNavShown = false;
 
+  const configPromise = fetch('config.json').then(response => response.json());
```

Comment:
Tää rakenne itseasiassa pitäis refaktoroida pois. Nyt meillä haetaan turhaan useammalla requestilla tuota config.jsonia kun tuo on päässyt lipumaan appiin, nav-baariin ja footeriin. Pitäisi keskitetysti tuolta app-kontrollerista valuttaa noille alikomponenteille. Koska sehän nyt vaan olisi järkevää.